### ISSUE TITLE
kfreebsd cannot modify $0

### ISSUE THREAD

**Issue by syohex** [kazeburo/Proclet#1]
This cause  01_run.t be failed.

```
syohei@kfreebsdi386:~/junk/perl$ uname -a
GNU/kFreeBSD kfreebsdi386 8.1-1-686 #0 Sat Jul 21 17:02:04 UTC 2012 i686 i386 Intel(R) Core(TM) i7-2600S CPU @ 2.80GHz GNU/kFreeBSD
syohei@kfreebsdi386:~/junk/perl$ cat prog_name.pl
#!/usr/bin/env perl
$0 = "hogehoge";
print `ps -p $$`;

syohei@kfreebsdi386:~/junk/perl$ ./prog_name.pl
  PID TTY          TIME CMD
54789 ?        00:00:00 perl
syohei@kfreebsdi386:~/junk/perl$ perl prog_name.pl
  PID TTY          TIME CMD
54795 ?        00:00:00 perl
```

'perldoc perlvar'  explain '$0' following:

```
On some (but not all) operating systems assigning to $0
modifies the argument area that the "ps" program sees.
```

Tests which modify $0 and use its value are not a good idea,
if you want to run Proclet on every platform.

**Comment by kazeburo** [comment]
Thank you for your big Hint.
I'll change 01_run.t  to not use $0.

**Comment by kazeburo** [comment]
Fixed in 0.08
